### DISCUSSION TITLE
is it possible to use make_geocube with paramter option all_touched = true?

### DISCUSSION THREAD

**Discussion by oswald1234** [corteva/geocube#97] [Q&A] [answered]
Is it possible to use make_geocube with parameter option all_touched = true?

**Comment by snowman2** [comment, marked as answer]
https://gis.stackexchange.com/a/427961/144357



You want to follow the pattern demonstrated here: https://corteva.github.io/geocube/stable/examples/rasterize_point_data.html
```python
from functools import partial
from geocube.rasterize import rasterize_image

roads = make_geocube(
    vector_data=roads,
    measurements=['train_id'],
    resolution=(10,-10),
    rasterize_function=partial(rasterize_image, all_touched=True),
)
```
Also see: https://corteva.github.io/geocube/stable/geocube.html#geocube.rasterize.rasterize_image

**Reply by oswald1234** [reply]
thank you!
